### PR TITLE
Make other things use modulo

### PR DESCRIPTION
**Pull request by MEEPofFaith**
Since genericCrafter now `%` instead of setting to 0, I figured other things should also do the same.

### CODE REVIEW

File: core/src/mindustry/world/blocks/power/ItemLiquidGenerator.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ public void updateTile(){
                     Item item = items.take();
                     productionEfficiency = getItemEfficiency(item);
                     explosiveness = item.explosiveness;
-                    generateTime = 1f;
+                    generateTime %= 1f;
```
Comment (Anuken) [inline core/src/mindustry/world/blocks/power/ItemLiquidGenerator.java:137]: yikes

Comment (MEEPofFaith) [reply core/src/mindustry/world/blocks/power/ItemLiquidGenerator.java:137]: -0.3 % 1 should = 0.7.
Or should I convert to `Mathf.mod` instead?

**Comment by MEEPofFaith** [comment]
oh dear

**Comment by MEEPofFaith** [comment]
https://user-images.githubusercontent.com/54301439/109405507-2d42c200-7926-11eb-9667-dd8bd46e4093.mp4

Somehow I can compile but not Travis.
Also, this is very much not right.

File: core/src/mindustry/world/blocks/power/ItemLiquidGenerator.java (hunk 1 above)
Comment (Anuken) [reply core/src/mindustry/world/blocks/power/ItemLiquidGenerator.java:137]: > -0.3 % 1 should = 0.7.

No, that's not how mod works, and there's no reason to use mod here at all.

**Comment by sk7725** [comment]
Modulo anly makes sense when a variable(n) is:
from 0 to a large float, += delta() each frame

some blocks have n go from 0 to 1, with a small float added each frame. Don not the modulo there.

**Comment by MEEPofFaith** [comment]
I'm lazy just gonna not bother.

**Pull request closed without merging**